src/logging.cpp

```diff
@@ -33,7 +33,7 @@ static int FileWriteStr(const std::string &str, FILE *fp)
 
 bool BCLog::Logger::StartLogging()
 {
-    std::lock_guard<std::mutex> scoped_lock(m_cs);
+    LockGuard scoped_lock(m_cs);
```

Comment:
16127 also should change line 266 from `std::lock_guard<std::mutex>` to `LockGuard`

Should also change four instances in stacktraces.cpp imo

`
src/stacktraces.cpp:173:    std::lock_guard<std::mutex> l(m);
src/stacktraces.cpp:597:    std::lock_guard<std::mutex> l(g_stacktraces_mutex);
src/stacktraces.cpp:606:    std::lock_guard<std::mutex> l(g_stacktraces_mutex);
src/stacktraces.cpp:670:    std::lock_guard<std::mutex> l(g_stacktraces_mutex);
`